src/Chainweb/Utils/RequestLog.hs

```diff
@@ -99,6 +101,11 @@ instance ToJSON RequestLog where
     toJSON = genericToJSON defaultOptions
         { fieldLabelModifier = over _head toLower . drop 11
         }
+    toEncoding = genericToEncoding defaultOptions
```

Comment:
Why do we still need generics?

Reply:
A lazy programer :-) The type is only used for telemetry logging.

Anyways, I'll add an explicit encoding for consistency.